adaptived/src/effects/sd_bus_setting.c

```diff
@@ -332,8 +354,14 @@ int sd_bus_setting_main(struct adaptived_effect * const eff)
 			ret = adaptived_get_meminfo_field(PROC_MEMINFO, "MemTotal", &ll_value);
 			if (ret)
 				return ret;
+
+			cgflags |= ADAPTIVED_CGROUP_FLAGS_VALIDATE;
```

Comment:
Should this be
```
cgflags = ADAPTIVED_CGROUP_FLAGS_VALIDATE;
```

What would be the use case where we want to `or` in previous values in cgflags?

Reply:
As currently implemented, it probably doesn't matter whether we use `=` or `|=`, but I'm wondering if in the future we could get bitten by erroneously or'ing in old data.

Reply:
Made the change.